freeipa/src/main/java/com/sequenceiq/freeipa/converter/cloud/StackToCloudStackConverter.java

```diff
@@ -403,6 +405,10 @@ private InstanceStatus getInstanceStatus(InstanceMetaData metaData, Collection<S
             params.put(RESOURCE_GROUP_NAME_PARAMETER, resourceGroupName);
             params.put(RESOURCE_GROUP_USAGE_PARAMETER, resourceGroupUsage.name());
         }
+        Optional<String> diskEncryptionSetId = getDiskEncryptionSetIdFromEnv(environmentCrn);
+        if (diskEncryptionSetId.isPresent()) {
+            params.put(DISK_ENCRYPTION_SET_ID, diskEncryptionSetId);
+        }
```

Comment:
Not required. Instance template attributes (containing CMK encryption type and DES ID) will be already pulled and populated in `buildInstanceTemplate()`.